client/app.jsx

```diff
@@ -0,0 +1,60 @@
+const React = require('react');
+const axios = require('axios');
+
+const OverviewTitle = require('./components/Title.jsx');
+const OverviewReview = require('./components/ReviewRating.jsx');
+const OverviewServices = require('./components/TypeofServices.jsx');
+const OverviewLastUpdated = require('./components/UpdateChecker.jsx');
+
+const OverviewUIbuttons = require('./components/UI_buttons.jsx');
+
+const baseURL = 'http://localhost:3001';
+
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      bizInfo: {},
+      loadComplete: false
+    };
+    // this.getBizInfo = this.getBizInfo.bind(this);

```

Comment:
Probably unnecessary code here since you're calling it in scope.

Reply:
I just put it as a placeholder tbh.
I will fix it and merge after.

Thanks